src/app/containers/InlineContainer/__snapshots__/index.test.jsx.snap

```diff
@@ -9,5 +27,13 @@ exports[`InlineContainer should render correctly 1`] = `
 <span
   className="c0"
   lang="fa"
-/>
+>
+  <a
+    aria-label="چیسربرگرundefined"
```

Comment:
any ideas where this 'undefined' is coming from?

Reply:
externalLinkText was missing.